tests/test_endpoint.py

```diff
@@ -14,12 +14,12 @@ def setUp(self):
         self.mock_ok_elasticsearch()
         self.expected_data = self.get_fixture('ok.json')
 
-    def assert_content(self, content, status='ok', **overrides):
-        data = json.loads(content)
+    def assert_response_content(self, response, status='ok', **overrides):
+        encoding = getattr(response, 'charset', 'utf-8')
```

Comment:
We're supporting two different behaviors: one for UTF-8 and one for who-knows-what. Should we use a concrete value instead so we know the exact behavior?